src/main/python/RLBotFramework/agents/base_flatbuffer_agent.py

```diff
@@ -0,0 +1,65 @@
+import flatbuffers
+from RLBotFramework.agents.base_agent import BaseAgent
+from RLBotMessages.flat import ControllerState
+from RLBotMessages.flat import PlayerInput
+from RLBotMessages.flat import GameTickPacket
+
+
+class SimpleControllerState:
+    """
+    Building flatbuffer objects is verbose and error prone. This class provides a friendlier
+    interface to bot makers.
+    """
+    def __init__(self):
+        self.steer = 0.0
+        self.throttle = 0.0
+        self.pitch = 0.0
+        self.yaw = 0.0
+        self.roll = 0.0
+        self.jump = False
+        self.boost = False
+        self.handbrake = False
+
+
+class BaseFlatbufferAgent(BaseAgent):
```

Comment:
if this is the way we decide to go and remove ctypes we should merge some of this into BaseAgent

Reply:
Agreed.